### PR TITLE
add(docker/hive):add hadoop aws jar

### PR DESCRIPTION
**Pull request by ArielCabib**

### CODE REVIEW

File: docker/hive/start-hive.sh [hunk 1]
```diff
@@ -50,6 +50,30 @@ cat >${HIVE_HOME}/conf/hive-site.xml <<EOL
...
+     <property>
+        <name>fs.s3.awsAccessKeyId</name>
+        <value>${AWS_ACCESS_KEY}</value>
+     </property>
+     <property>
+        <name>fs.s3.awsSecretAccessKey</name>
+        <value>${AWS_SECRET_KEY}</value>
+     </property>
+     <property>
+        <name>hive.server2.thrift.sasl.qop</name>
```
Comment (lyogev) [inline docker/hive/start-hive.sh]: does this solve the SASL errors?

File: docker/hive/start-hive.sh [hunk 2]
```diff
@@ -50,6 +50,30 @@ cat >${HIVE_HOME}/conf/hive-site.xml <<EOL
         <name>fs.default.name</name>
         <value>${DEFAULT_FS}</value>
      </property>
+     <property>
+        <name>hive.metastore.sasl.enabled</name>
```
Comment (lyogev) [inline docker/hive/start-hive.sh]: does this solve the SASL errors?

File: docker/hive/Dockerfile [hunk 3]
```diff
@@ -22,6 +22,14 @@ ENV HUDI_VERSION=0.4.5
 RUN apt-get update && apt-get install -y ant
 RUN wget -q http://central.maven.org/maven2/com/uber/hoodie/hoodie-hive-bundle/$HUDI_VERSION/hoodie-hive-bundle-$HUDI_VERSION.jar \
     && mv hoodie-hive-bundle-$HUDI_VERSION.jar $HIVE_HOME/lib
+RUN wget -q http://central.maven.org/maven2/org/apache/hadoop/hadoop-aws/$HADOOP_VERSION/hadoop-aws-$HADOOP_VERSION.jar \
+    && mv hadoop-aws-$HADOOP_VERSION.jar $HIVE_HOME/lib
+RUN wget -q http://central.maven.org/maven2/com/amazonaws/aws-java-sdk/1.7.4/aws-java-sdk-1.7.4.jar \
+    && mv aws-java-sdk-1.7.4.jar $HIVE_HOME/lib
+RUN wget -q http://central.maven.org/maven2/net/logstash/log4j/jsonevent-layout/1.7/jsonevent-layout-1.7.jar \
+    && mv jsonevent-layout-1.7.jar $HIVE_HOME/lib
+RUN wget -q http://central.maven.org/maven2/net/minidev/json-smart/1.1.1/json-smart-1.1.1.jar \
```
Comment (lyogev) [inline docker/hive/Dockerfile]: If you're adding these JARs please also handle log4j.properties to use them

File: docker/hive/start-hive.sh (hunk 1 above)
Comment (ArielCabib) [reply docker/hive/start-hive.sh]: nope

File: docker/hive/start-hive.sh (hunk 2 above)
Comment (ArielCabib) [reply docker/hive/start-hive.sh]: nope

File: docker/hive/start-hive.sh [hunk 4]
```diff
@@ -50,6 +60,34 @@ cat >${HIVE_HOME}/conf/hive-site.xml <<EOL
...
+     </property>
+     <property>
+        <name>fs.s3a.impl</name>
+        <value>org.apache.hadoop.fs.s3a.S3AFileSystem</value>
+     </property>
+     <property>
+        <name>fs.s3.impl</name>
+        <value>org.apache.hadoop.fs.s3a.S3AFileSystem</value>
+     </property>
+     <property>
```
Comment (lyogev) [inline docker/hive/start-hive.sh]: Can you make these 2 properties optional?

File: docker/hive/start-hive.sh (hunk 1 above)
Comment (lyogev) [reply docker/hive/start-hive.sh]: We can remove it I guess

File: docker/hive/start-hive.sh (hunk 2 above)
Comment (lyogev) [reply docker/hive/start-hive.sh]: we can remove it I guess

File: docker/hive/start-hive.sh [hunk 5]
```diff
@@ -7,6 +7,16 @@ METASTORE_PORT=${METASTORE_PORT:=9083}
 DEFAULT_FS=${DEFAULT_FS:=file:///}
 DB_TYPE=${DB_TYPE:=mysql}
 
+cat >${HIVE_HOME}/conf/hive-log4j.properties <<EOL
+log4j.rootLogger=INFO, CONSOLE
+
+# Disable excessive reflection warnings - KAFKA-5229
```
Comment (lyogev) [inline docker/hive/start-hive.sh]: Add a file to the docker log4j.json then copy it here if you got use_json_logger env variable

File: docker/hive/start-hive.sh [hunk 6]
```diff
@@ -7,6 +7,16 @@ METASTORE_PORT=${METASTORE_PORT:=9083}
...
+log4j.logger.org.reflections=ERROR
```
Comment (lyogev) [inline docker/hive/start-hive.sh]: Do we need this?

File: docker/hive/start-hive.sh (hunk 4 above)
Comment (ArielCabib) [reply docker/hive/start-hive.sh]: @lyogev how exactly? if they're empty, we'll get empty entries

File: docker/hive/start-hive.sh (hunk 5 above)
Comment (ArielCabib) [reply docker/hive/start-hive.sh]: @lyogev i dont understand

File: docker/hive/start-hive.sh (hunk 6 above)
Comment (ArielCabib) [reply docker/hive/start-hive.sh]: @lyogev i copied it from cdc: https://github.com/YotpoLtd/change-data-capture

**Pull request merged**